static/src/javascripts/projects/common/modules/commercial/close-disabled-slots.js

```diff
@@ -0,0 +1,58 @@
+define([
+    'bonzo',
+    'qwery',
+    'Promise',
+    'common/utils/$css',
+    'common/utils/fastdom-promise',
+    'common/modules/commercial/commercial-features'
+], function (
+    bonzo,
+    qwery,
+    Promise,
+    $css,
+    fastdom,
+    commercialFeatures
+) {
+    var adSlotSelector = '.js-ad-slot';
+
+    return {
+        init: init
```

Comment:
For what purpose did you move this in a separate module? This feels over-engineered, especially since this module will linger in run once and then linger in memory forever.

fwiw, I keep a log of all incoming slots and mark them as hidden when necessary, in [my refactoring of DFP](https://github.com/guardian/frontend/pull/13289/files#diff-844a43280e1947d8dd7cc7a480fd0551R336). From there, we could reassign/refresh an existing slot manually, e.g. `top-above-nav'—removing it from the DOM does not remove it from memory, although for now I have taken care of it.